.travis.yml

```diff
@@ -3,5 +3,4 @@ rvm:
   - 2.0.0
   - 1.9.3
   - jruby-19mode
-  - rbx-19mode
```

Comment:
Since this is an open-source project and we don't know who's using it or with what Ruby, I'd probably keep this in and do something like:

```yaml
matrix:
  allow_failures:
  - rvm: rbx-19mode
```